android/src/main/java/com/swmansion/reanimated/NativeMethodsHelper.java

```diff
@@ -64,6 +72,15 @@ public static void scrollTo(View view, double argX, double argY, boolean animate
 
   }
 
+  private static ReactScrollView findScrollView(ReactSwipeRefreshLayout view) {
+    for(int i = 0; i < view.getChildCount(); i++) {
+      if(view.getChildAt(i) instanceof ReactScrollView) {
```

Comment:
Is it possible to have HorizontalScrollView here?

Reply:
ReactSwipeRefreshLayout support only vertical scroll - "[The SwipeRefreshLayout should be used whenever the user can refresh the contents of a view via a vertical swipe gesture.](https://developer.android.com/reference/androidx/swiperefreshlayout/widget/SwipeRefreshLayout)"